source/index.txt

```diff
@@ -13,5 +13,59 @@ MongoDB C# Driver
    /compatibility
    /issues-and-help
    /faq
-   
 
+Introduction
+------------
+
+Welcome to the documentation site for the official {+driver-long+}. 
+You can add the driver to your application to work with MongoDB in {+language+}. 
+Download it using `NuGet <https://www.nuget.org/>`__, or set up a runnable
```

Comment:
Issue: the use of "it" here is unclear as you also mentioned a few other nouns in the previous sentence.

```suggestion
Download the driver using `NuGet <https://www.nuget.org/>`__, or set up a runnable
```